rtl/__init__.py

```diff
@@ -465,27 +476,33 @@ def rtlcmd(self):
         gstring=' '.join([ ('-g ' + str(param) +'='+ str(val)) 
             for param,val in iter(self.rtlparameters.items()) ])
 
+        vlogsimargs = ' '.join(self.vlogsimargs)
+
         fileparams=''
         for name, file in self.iofile_bundle.Members.items():
             fileparams+=' '+file.simparam
 
+        dofile=self.interactive_controlfile
```

Comment:
Assigns `interactive controlfile` regardless of `self.interactive_rtl`. This is to allow generating a waveform even if you don't want to open it immediately. Useful when running dozens of tests, from which you don't beforehand know which one fails.